.github/workflows/ci.yml

```
@@ -63,3 +63,37 @@ jobs:
+  build-test-windows:
+    runs-on: windows-latest
+    permissions:
+      contents: read
+    env:
+      CARGO_TERM_COLOR: always
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install rust stable
+        uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af
+        with:
+            toolchain: stable
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 10 but found 12

(indentation)